beacon_node/rest_api/src/validator.rs

```diff
@@ -0,0 +1,149 @@
+use super::{success_response, ApiResult};
+use crate::{helpers::*, ApiError, UrlQuery};
+use beacon_chain::{BeaconChain, BeaconChainTypes};
+use bls::PublicKey;
+use hyper::{Body, Request};
+use serde::{Deserialize, Serialize};
+use std::sync::Arc;
+use store::Store;
+use types::beacon_state::EthSpec;
+use types::{BeaconBlock, BeaconState, Epoch, RelativeEpoch, Shard, Slot};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ValidatorDuty {
+    /// The validator's BLS public key, uniquely identifying them. _48-bytes, hex encoded with 0x prefix, case insensitive._
+    #[serde(rename = "validator_pubkey")]
```

Comment:
What's the deal with these `renames`? Does serde to `validator-pubkey`?

Reply:
No, these were automatically generated from the OpenAPI

Reply:
They aren't necessary and I will remove them.